tests/integration_tests/sqllab_tests.py

```diff
@@ -406,22 +407,18 @@ def test_search_query_on_text(self):
         self.assertEqual(2, len(data))
         self.assertIn("birth", data[0]["sql"])
 
-    def test_search_query_on_time(self):
+    def test_search_query_filter_by_time(self):
         self.run_some_queries()
         self.login("admin")
         first_query_time = (
             db.session.query(Query).filter_by(sql=QUERY_1).one()
         ).start_time
         second_query_time = (
-            db.session.query(Query).filter_by(sql=QUERY_3).one()
+            db.session.query(Query).filter_by(sql=QUERY_2).one()
         ).start_time
-        # Test search queries on time filter
-        from_time = "from={}".format(int(first_query_time))
-        to_time = "to={}".format(int(second_query_time))
-        params = [from_time, to_time]
-        resp = self.get_resp("/superset/search_queries?" + "&".join(params))
-        data = json.loads(resp)
-        self.assertEqual(2, len(data))
+        # pylint: disable=line-too-long
```

Comment:
I would add this at the end of line #420, otherwise I think it disables the error for the entire block.

Reply:
I reformatted it so pylint disable isn't necessary.